src/mdb_v8.c

```diff
@@ -3362,6 +3444,60 @@ jsobj_print_jsarray(uintptr_t addr, jsobj_print_t *jsop)
 }
 
 static int
+jsobj_print_jstyped_array(uintptr_t addr, jsobj_print_t *jsop)
```

Comment:
Are TypedArrays used for anything other than Buffers?  Would it be reasonable to have this code path print something like: "<TypedArray of length N>", rather than trying too hard to decode it?  Basically, as it is now, I'm not sure what this really _should_ do, but for the reasons you mentioned before, I'm reluctant to pretend like it has a "length" property that it doesn't really have.